pkg/generated/openapi/openapi_test.go

```diff
@@ -23,12 +23,13 @@ import (
 
 	"github.com/go-openapi/spec"
 
+	generatedopenapi "k8s.io/apiextensions-apiserver/pkg/generated/openapi"
 	"k8s.io/apimachinery/pkg/util/diff"
 )
 
 func TestOpenAPIRoundtrip(t *testing.T) {
 	dummyRef := func(name string) spec.Ref { return spec.MustCreateRef("#/definitions/dummy") }
-	for name, value := range GetOpenAPIDefinitions(dummyRef) {
+	for name, value := range generatedopenapi.GetOpenAPIDefinitions(dummyRef) {
```

Comment:
why is this changing... this is currently testing the generated openapi under pkg/generated/openapi/zz_generated.openapi.go

Reply:
When I click on GetOpenAPIDefinitions in the test (with this change), it brings me to ./staging/src/k8s.io/apiextensions-apiserver/pkg/generated/openapi/zz_generated.openapi.go

Same with the other test.

Isn't this what you expected, @liggitt